### ISSUE TITLE
Recursive link in README

### ISSUE THREAD

**Issue by riban-bw** [dxinteractive/ResponsiveAnalogRead#30]
The README has a link to http://damienclarke.me/code/posts/writing-a-better-noise-reducing-analogread which redirects back to GitHub hence is of little use. Is the text of the referred article available? Maybe it could be added to the GitHub wiki.

**Comment by davidedelvento** [comment]
Agreed. Regarding reading the article, the internet archive comes to the rescue: https://web.archive.org/web/20170409184051/http://damienclarke.me/code/posts/writing-a-better-noise-reducing-analogread
